pkg/build/nodeimage/defaults.go

```diff
@@ -22,4 +22,4 @@ const DefaultImage = "kindest/node:latest"
 // DefaultBaseImage is the default base image used
 // TODO: come up with a reasonable solution to digest pinning
 // https://github.com/moby/moby/issues/43188
-const DefaultBaseImage = "docker.io/kindest/base:v20241212-9f82dd49"
+const DefaultBaseImage = "docker.io/kindest/base:v20250117-f528b021"
```

Comment:
can we leave this out of this commit? this will merge in another commit, see discussion in https://github.com/kubernetes-sigs/kind/pull/3848

Reply:
(also, I would expect the PR to work with the old base image / node images anyhow?? this is nerdctl on the host, right?, github actions doesn't build node images currently)

Reply:
Yes, sure.